tests/shipment_test.go

```diff
@@ -374,3 +374,21 @@ func (c *ClientTests) TestShipmentReRateWithCarbonOffset() {
 	assert.Nil(baseRate.CarbonOffset)
 	assert.NotNil(newRateWithCarbon.CarbonOffset)
 }
+
+func (c *ClientTests) TestBuyShipmentWithEndShipper() {
+	c.T().Skip("Skipping because this requires production credentials and would spend real money")
+
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	shipment, err := client.CreateShipment(c.fixture.OneCallBuyShipment())
```

Comment:
If you create a shipment with the one-call buy fixture, it'll buy it making this test blow up since you can't buy an already bought shipment.

```suggestion
	shipment, err := client.CreateShipment(c.fixture.BasicShipment())
```

Reply:
Resolved